### ISSUE TITLE
Question about GRPO Training Data

### ISSUE THREAD

**Issue by uwGZQ** [wdrink/SimpleAR#17]
Great work, very impressive! 
I was wondering if you could kindly share more details about the data used during the GRPO training phase:

1. What was the source of the data during GRPO training?
2. What was the approximate scale? (number of samples)

Thanks in advance!

**Comment by wdrink** [comment]
Hi @uwGZQ , thanks for your interest in our work. We sampled approximately 11k prompts from the SFT training data to perform GRPO training. 

As mentioned in the paper, during SFT, we use both short captions and long captions to facilitate our model to generate high-quality images given both types of prompts. During RL stage, we tried to sample only long or only short prompts. The results showed that both were ok and using only short prompts led to better performance. 

Feel free to reach out if you have any other questions, I'm quite glad to provide help to your reproduction!

**Comment by uwGZQ** [comment]
Thanks for your reply! 
I also wonder how many GPUs (and of what model) were used during GRPO training, and how long did this training take?

**Comment by wdrink** [comment]
I used 8 A100 GPUs and it took around 1 day.

**Comment by uwGZQ** [comment]
Very helpful, thanks again.

**Comment by uwGZQ** [comment]
A follow up question:
I noticed that in the README you mentioned that the GRPO training involved careful parameter tuning. Could you tell me which parameters have the most significant impact on the model's performance and how your team has fine-tuned them?

**Comment by wdrink** [comment]
Below are our findings:

1.  temperature = 1.0 leads to better results than temperature < 1.0
2. warm up could make the early training more stable, but disabling it led to similar results (you may try it on your data)
3. we didn't tune the learning rate a lot, and under our setting, increasing the learning rate to 1e-5 may improve the performance a bit.

**Comment by uwGZQ** [comment]
Thanks!

**Comment by zghhui** [comment]
hi, this is a good work😀, I have some other questions:
- What is the batch_size and group_size you use during GRPO training?
- How do these two parameters affect the performance?
